static/js/containers/TermsOfServicePage.js

```diff
@@ -31,25 +36,147 @@ class TermsOfServicePage extends React.Component {
   }
 
   render() {
-    return <div>
-      <h3>
-        MIT Micromasters Terms of Service
-      </h3>
-
-      <p>
-        Please read through the terms of service. To enroll in MIT Micromasters you
-        must agree to the terms of service.
-      </p>
+    const profile = this.props.userProfile.profile;
+    let linkToEdxTermsOfService = (
+      <a href="https://www.edx.org/edx-terms-service" target='_blank'>https://www.edx.org/edx-terms-service</a>
+    );
+    let imageUrl = `${SETTINGS.edx_base_url}/static/images/profiles/default_120.png`.
+    //replacing multiple "/" with a single forward slash, excluding the ones following the colon
+    replace(/([^:]\/)\/+/g, "$1");
+    if (profile.profile_url_large) {
+      imageUrl = profile.profile_url_large;
+    }
+    return (<div className="card">
+        <Grid className="card-user">
+        <Cell col={2} className="card-image-box">
+            <UserImage imageUrl={imageUrl}/>
+        </Cell>
+        <Cell col={5} className="card-name">
+          { profile.preferred_name || SETTINGS.name }
+          <div className="card-student-id">
+            ID: { profile.pretty_printed_student_id }
+          </div>
+        </Cell>
+      </Grid>
+      <div className="card-copy terms-of-service-main">
+        <div className="program">
+          <div className="terms-card-header">
+            Terms of service
+          </div>
+          <div className="course-list terms-of-service-container">
+            <div className="course-list-header course-list-center course-list-row"
+              style={{height: DASHBOARD_COURSE_HEIGHT}}>
+              <h4 className="course-title">Please read and agree to our Terms of Service</h4>
```

Comment:
This isn't going to be the right className. We seem to be all over the place with these right now. 

The styling should be the same as https://github.com/mitodl/micromasters/blob/master/static/scss/dashboard.scss#L212-L214

Let me suggest terms-of-service-title for consistency, although I think these might all be the same style.